cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractStructuredMetadataTest.java

```diff
@@ -27,6 +27,8 @@ public static void setUpClass() throws IOException {
         if (cloudinary.config.apiSecret == null) {
             System.err.println("Please setup environment for Upload test to run");
         }
+
+        cloudinary.uploader().upload(SRC_TEST_IMAGE, asMap("public_id", PUBLIC_ID));
```

Comment:
Is that needed because of the test-cloud change? no "sample" in that cloud?

Reply:
Yes. This can probably be fixed in the cloud somehow however we agreed it's more reliable to not have any assumptions about pre-existing resources. Sample can be deleted even after it's created so it wasn't a great idea to begin with...